arrocke/gloss-translation
# feat: improve loading indicator

## What has changed



Replaced loading text with loading spinner in translation view.

## Connected Issues



Closes #103 

## QA Steps



1. Go to the translation page.
2. Click one of the arrows to switch verses.

The loading indicator should be shown. The verse selector should not disappear.

## Post-Deployment
